Modules/Bridge/VtkGlue/wrapping/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 itk_wrap_module(ITKVtkGlue)
-  if(PYTHON_VERSION_STRING VERSION_GREATER 3.0 AND "${VTK_VERSION}" VERSION_LESS 7.0.0)
+  if("${VTK_VERSION}" VERSION_LESS 7.0.0)
     message(WARNING
       "The ITKVtkGlue module can only built with Python 3 and VTK >= 7.\n"
```

Comment:
The error message should be updated as well.

Reply:
The error message is still correct, although redundant in stating that Python 3 is required.  In this case, I chose to leave the redundant documemtation.